### PR TITLE
Bugfix 557

### PR DESCRIPTION
**Pull request by ziajka**
Possible solution for the IOU problem.

### CODE REVIEW

File: gns3server/schemas/iou.py [hunk 1]
```diff
@@ -182,7 +186,11 @@
         "command_line": {
             "description": "Last command line used by GNS3 to start QEMU",
             "type": "string"
-        }
+        },
+        "application_id": {
+            "description": "Application ID for running IOU image",
+            "type": ["integer", "null"]
```
Comment (noplay) [inline gns3server/schemas/iou.py]: I think application_id in  IOU_OBJECT_SCHEMA  can't be null

File: tests/controller/test_project.py [hunk 2]
```diff
@@ -383,7 +361,7 @@ def test_deleteDrawing(async_run, project, controller):
     assert len(project._drawings) == 0
 
 
-def test_cleanPictures(async_run, project, controller):
+def test_clean_pcictures(async_run, project, controller):
```
Comment (noplay) [inline tests/controller/test_project.py]: Typo for pictures

File: gns3server/schemas/iou.py (hunk 1 above)
Comment (ziajka) [inline gns3server/schemas/iou.py]: We support old behavior (local generation) still, but of course we can force to define application_id.

Comment (noplay) [inline gns3server/schemas/iou.py]: If I'm correct the OBJECT is not use for creation but just when we return the info. In this case the application_id should have been generated

Comment (ziajka) [inline gns3server/schemas/iou.py]: Good point! Thank you.

**Pull request merged**